packages/ui/src/components/ListV2/__stories__/Perf.stories.tsx

```diff
@@ -0,0 +1,79 @@
+import type { Story } from '@storybook/react'
+import { useMemo, useState } from 'react'
+import { List } from '..'
+import { Button, PaginationV2, Stack } from '../..'
+
+const ITEM_PER_PAGE = 50
```

Comment:
Can we reduce to 25 per pages? Or is it on purpose to show performance? This seems quite a long story

Reply:
yes I think we can delete the story itself before merginig, it was more for checking perfs